src/rgb/resolvers.rs

```diff
@@ -27,14 +25,20 @@ impl rgb::Resolver for ExplorerResolver {
         scripts: BTreeMap<DeriveInfo, bitcoin_30::ScriptBuf>,
     ) -> Result<BTreeSet<rgb::prelude::Utxo>, String> {
         let mut utxos = bset![];
-        let explorer_client = EsploraBlockchain::new(&self.explorer_url, 100);
+        // TODO: Find a way to run async function synchronously (wasm32)
```

Comment:
@cryptoquick and @josediegorobles, 

here is the place I use the esplora-sync.

Reply:
That's really strange, it supposed to be running the blocking version yet. I will try if i can figure out myself something.